apps/api/src/campaign/campaign.controller.ts

```diff
@@ -54,11 +54,18 @@ export class CampaignController {
     @Body() createDto: CreateCampaignDto,
     @AuthenticatedUser() user: KeycloakTokenParsed,
   ) {
-    let person
     if (!isAdmin(user)) {
+      Logger.warn('The campaign creator is not in admin role. User name is: ' + user.name)
+    }
+
+    let person
+    if (!createDto.coordinatorId) {
+      //Find the creator personId and make him a coordinator
       person = await this.personService.findOneByKeycloakId(user.sub as string)
+
       if (!person) {
-        throw new NotFoundException('No person found for logged user')
+        Logger.error('No person found in database for logged user: ' + user.name)
```

Comment:
The field `email` is also unique both in our db and keycloak db, but it might be considered sensitive data.
I do agree we can improve the log, but that should not be a blocker